chingu-voyages/v37-bears-team-14
# Add kafka messaging for queueing notifications to push to the frontend

# Changes
- Adds kafka producer and consumers to queue notifications for the frontend
- Adds event listening to the useNotifications hook

# Testing
- Update `.env.dev` configs with kafka credentials https://discord.com/channels/330284646283608064/939665645107282010/940802454881194046
- Run app
- Go to `/notifications` page
- Open a second window and go to `/admin/notifications`
- Send a test message, wait a few seconds, and confirm that the event shows up in the notification list.

Closes #131 
Closes #132
